promql/testdata/functions.test

```diff
@@ -734,3 +843,69 @@ eval instant at 5m absent_over_time({job="ingress"}[4m])
 
 eval instant at 10m absent_over_time({job="ingress"}[4m])
 	{job="ingress"} 1
+
+clear
+
+# Testing exp() sqrt() log2() log10() ln()
+load 5m
+	vector_matching_a{l="x"} 10
+	vector_matching_a{l="y"} 20
+	vector_matching_b{l="x"} 40
```

Comment:
This is unused (and the metric name can be simplified, we're not doing vector matching).